vue.config.js

```diff
@@ -2,10 +2,11 @@ const zlib = require('zlib');
 const PrerenderSPAPlugin = require('prerender-spa-plugin');
 const path = require('path');
 module.exports = {
-    devServer: {
-        host: 'parkspot.localhost',
-        proxy: 'https://www.parkspot.in',
-    },
+    // was working but now not working why?
```

Comment:
do we need this change?

Reply:
it was needed when we are not able to test firebase in local due to authorised domain... now I have whitelisted localhost ..so not needed.. also it stopped working as well.